content/docs/next-release/operator.md

```diff
@@ -202,11 +208,28 @@ spec:
 ```
 <1> Identifies the Kafka configuration used by the collector, to produce the messages, and the ingester to consume the messages.
 
-<2> The deadlock interval can be disabled to avoid the ingester being terminated when no messages arrive within the default 1 minute period
+<2> The deadlock interval is disabled by default (set to 0), to avoid the ingester being terminated when no messages arrive, but can be configured to specify the number of minutes to wait for a message before terminating.
 
-{{< info >}}
-A Kafka environment can be configured using [Strimzi's Kafka operator](https://strimzi.io/).
-{{< /info >}}
+#### Self Provisioned Kafka Cluster
+
+To use the self-provisioned approach, the producer/consumer brokers property should not be defined.
+
+```yaml
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: auto-provision-kafka
+spec:
+  strategy: streaming
+  storage:
+    type: elasticsearch
+    options:
+      es:
+        # Note: This assumes elasticsearch is running in the "default" namespace.
+        server-urls: http://elasticsearch.default.svc:9200
+```
+
+The self-provision of a Kafka cluster can be disabled by setting the flag `--kafka-provision` to `false`. The default value is `auto`, which will make the Jaeger Operator query the Kubernetes cluster for its ability to handle a `Kafka` custom resource. This is usually set by the Kafka Operator during its installation process, so, if the Kafka Operator is expected to run *after* the Jaeger Operator, the flag can be set to `true`.
```

Comment:
This is more or less a copy of the ES equivalent paragraph - assume it is valid?